src/Microsoft.Identity.Client/Instance/AdfsAuthority.cs

```diff
@@ -48,7 +48,7 @@ public AdfsAuthority(IServiceBundle serviceBundle, AuthorityInfo authorityInfo)
 
         internal override string GetTenantId()
```

Comment:
Please add some comments on what it means to return null, i.e. why ADFS does not support multiple tenants etc.